src/containers/Tasks/Tasks.js

```diff
@@ -287,45 +286,13 @@ export /* istanbul ignore next */ class Tasks extends Component {
           )}
         />
         {showDeleteModal ? (
-          <Modal
-            open={showDeleteModal}
-            primaryButtonText={intl.formatMessage({
-              id: 'dashboard.actions.deleteButton',
-              defaultMessage: 'Delete'
-            })}
-            secondaryButtonText={intl.formatMessage({
-              id: 'dashboard.modal.cancelButton',
-              defaultMessage: 'Cancel'
-            })}
-            modalHeading={intl.formatMessage(
-              {
-                id: 'dashboard.deleteResources.heading',
-                defaultMessage: 'Delete {kind}'
-              },
-              { kind: 'Tasks' }
-            )}
-            onSecondarySubmit={this.closeDeleteModal}
-            onRequestSubmit={this.handleDelete}
-            onRequestClose={this.closeDeleteModal}
-            danger
-          >
-            <p>
-              {intl.formatMessage(
-                {
-                  id: 'dashboard.deleteResources.confirm',
-                  defaultMessage:
-                    'Are you sure you want to delete these {kind}?'
-                },
-                { kind: 'Tasks' }
-              )}
-            </p>
-            <UnorderedList nested>
-              {toBeDeleted.map(task => {
-                const { name, namespace } = task.metadata;
-                return <ListItem key={`${name}:${namespace}`}>{name}</ListItem>;
-              })}
-            </UnorderedList>
-          </Modal>
+          <DeleteModal
+            kind="Tasks"
```

Comment:
Dont we have constants somewhere for the kind ? doesn't have to change, just if they're there might be better.

Reply:
There are no constants for these since they're usually only used in a single container, 'Tasks' for example is only used here in the Tasks container and in 1 test.